### PR TITLE
Added Note

### PR DESCRIPTION
**Pull request by mukeshHCW**
Note to redirect users specially for Government cloud like GCC, DoD etc.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeHybrid/hybrid-configuration-wizard.md [hunk 1]
```diff
@@ -87,6 +87,10 @@ The table below outlines the main options that the Hybrid Configuration wizard m
...
+
+> [!NOTE]
+> If the tenant is hosted in Office 365 U.S. Government GCC High or DoD environment,  the -MetadataUrl parameter in Set-FederationTrust would be different. Please refer to [Set-FederationTrust](https://technet.microsoft.com/library/dd298034\(v=exchg.150\)) for more information.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeHybrid/hybrid-configuration-wizard.md:92]:
```suggestion
> If the tenant is hosted in Office 365 U.S. Government GCC High or DoD environment, the -MetadataUrl parameter in Set-FederationTrust would be different. Please refer to [Set-FederationTrust](https://technet.microsoft.com/library/dd298034\(v=exchg.150\)) for more information. 
```

**Comment by mattpennathe3rd** [comment]
@scanum @JohanFreelancer9 No further work on this PR. It will be obsoleted by another.

**Comment by mukeshHCW** [comment]
@mattpennathe3rd : is something pending on me here..?

**Pull request closed without merging**